src/components/MainView/MainView.jsx

```diff
@@ -3,13 +3,49 @@ import { Container, Col, Row } from 'react-bootstrap';
 import '../../App.scss';
 import CreateDashboard from './CreateDashboard/CreateDashboard';
 import ListOfDashboards from './ListOfDashboards/ListOfDashboards';
+import { db } from '../../fire';
+import getDashArrayFromDb from './getDashArrayFromDb';
 import NavBar from '../Dashboard/Header/Header';
+import WelcomeView from '../Dashboard/Header/WelcomeView';
 
 class MainView extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoggedIn: props.isLoggedIn,
+      dashboards: [],
+    };
+  }
+
+  componentDidMount() {
```

Comment:
Please see the development branch for current code of this file and update this pull-request accordingly

Reply:
Please, provide more information on what is wrong with this file.

Reply:
You are trying to build your code upon the outdated version of mine.